build.gradle

```diff
@@ -160,10 +159,10 @@ dependencies {
         exclude group: 'org.matsim', module: 'matsim'
     }
 
-    compile(group: 'com.github.wrashid.matsim', name: 'matsim', version: '0.10.1-beam-6') {
+    compile ('com.github.wrashid.matsim:matsim:rajnikantsh~remove-redundant-code-SNAPSHOT'){
```

Comment:
Please, let's make proper release to make sure we don't miss anything

Reply:
@REASY this dependency change require due to compatible with specific branch in matsim and beam-utilities . This dependency is pointing to that specific branch in matsim(all 3 projects Beam, Beam-Utilities, Matsim needs to be in sync) . After final approval in all 3 we can change dependency.